docker-compose.yml

```diff
@@ -58,14 +58,16 @@ services:
       - celestia-zkevm-net
 
   rollkit-evm-single:
-    image: ghcr.io/rollkit/rollkit-evm-single:v0.1.3
+    image: ghcr.io/rollkit/rollkit-evm-single:pr-2394
     container_name: rollkit-evm-single
     entrypoint: /usr/bin/entrypoint.sh
+    volumes:
+      - evm-single-data:/data
```

Comment:
is the rollkit/sequencer block store maintained under a directory called `/data` at the root of the filesystem?

would be nice to put this into some standard home directory eventually, but it can be done later, would've execpted it at somewhere like `~/.rollkit/data`